src/genetic_functions/crossover.jl

```diff
@@ -0,0 +1,48 @@
+function crossover_2_children(parent1::RuleNode, parent2::RuleNode)
```

Comment:
lets name these more informatively. I like the format of keeping the role of the function within GA as the first part of the name: `crossover_X`, `mutate_X`, `select_X`.
So maybe we call this one `crossover_swap_children_2`?

Reply:
Yes I will rename the function name. But I am not sure if `crossover_swap_children_2` indicates that two children are returned.